submissions/NwagboJohn/my_php/index.php

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
+
+    <head>
+        <title>Buy More</title>
+        <link rel="stylesheet" type="text/css" href="index.css">
+    </head>
+
+    <body>
+        <h1> BUY MORE</h1>
+        <hr>
+        <div id="form">
+            <h1>UPLOAD STOCK</h1>
+            <form action="logic.php" method="post">
+                BRAND <input type="text" name="brands"><br><br>

```

Comment:
these brands should ideally come from the database and loaded into a dropdown. That way, you don't have to be doing the OR checks in your logic
```if ($brand_name == "nokia" || $brand_name == "infinix" || $brand_name == "samsung" || $brand_name == "techno" || $brand_name == "gionee" || $brand_name == "iphone") {```